src/goe/filesystem/web_hdfs.py

```diff
@@ -144,11 +143,12 @@ def __init__(
             % (hdfs_namenode, webhdfs_port, hdfs_user, use_kerberos, verify_ssl_cert)
         )
 
-        super(WebHdfs, self).__init__(messages, dry_run=dry_run)
+        super(WebHdfs, self).__init__(
+            messages, dry_run=dry_run, do_not_connect=do_not_connect
+        )
 
         self._db_path_suffix = db_path_suffix
         self._hdfs_data = hdfs_data
-        self._do_not_connect = do_not_connect
```

Comment:
Another `_do_not_connect` removed. I assume it's not needed, but just checking...

Reply:
Correct, the parent has the same line, this one was duplication.